velox/exec/tests/AggregationFuzzerTest.cpp

```diff
@@ -43,6 +44,11 @@ DEFINE_string(
     "this comma separated list of function names "
     "(e.g: --only \"min\" or --only \"sum,avg\").");
 
+DEFINE_string(
+    prestoCoordinatorUri,
```

Comment:
consistency: presto_url (let's not include coordinator as it makes it very long)